libraries/AP_Param/AP_Param.h

```diff
@@ -568,7 +568,9 @@ class AP_Param
 
     // return the persistent top level key for the ParamToken key
     static uint16_t get_persistent_key(uint16_t key) { return var_info(key).key; }
-    
+
+    bool allow_set_via_mavlink(uint16_t flags) const;
```

Comment:
maybe we can put a comment above this function too?

Reply:
Done in precursor PR which will be merged before this one.